frontend/app-development/layout/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { type ReactElement } from 'react';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { StudioPageHeader, useMediaQuery } from '@studio/components';
+import { useRepoMetadataQuery } from 'app-shared/hooks/queries';
+import { SubHeader } from './SubHeader';
+import { MEDIA_QUERY_MAX_WIDTH } from 'app-shared/constants';
+import { SmallHeaderMenu } from './SmallHeaderMenu';
+import { type HeaderMenuItem } from 'app-development/types/HeaderMenu/HeaderMenuItem';
+import { useTranslation } from 'react-i18next';
+import { LargeNavigationMenu } from './LargeNavigationMenu';
+import { usePageHeaderContext } from 'app-development/contexts/PageHeaderContext';
+import { UserProfileMenu } from './UserProfileMenu';
+
+export type PageHeaderProps = {
+  showSubMenu: boolean;
+  isRepoError?: boolean;
+};
+
+export const PageHeader = ({ showSubMenu, isRepoError }: PageHeaderProps): ReactElement => {
+  const { t } = useTranslation();
+  const { app } = useStudioEnvironmentParams();
+  const { menuItems } = usePageHeaderContext();
+
+  const shouldDisplaySmallMenu = useMediaQuery(MEDIA_QUERY_MAX_WIDTH);
+  const shouldDisplayText = !shouldDisplaySmallMenu;
+
+  return (
+    <StudioPageHeader>
+      <StudioPageHeader.Main>
+        <StudioPageHeader.Left showTitle={shouldDisplayText} title={app} />
+        {!shouldDisplaySmallMenu && (
+          <StudioPageHeader.Center>
+            {menuItems && (
+              <LargeNavigationMenu
+                menuItems={menuItems.map((menuItem: HeaderMenuItem) => ({
+                  link: menuItem.link,
+                  name: t(menuItem.key),
+                  isBeta: menuItem.isBeta,
+                }))}
+              />
+            )}
+          </StudioPageHeader.Center>
+        )}
```

Comment:
Would the code be easier to read if we change the name of `shouldDisplaySmallMenu ` to `shouldDisplayDesktopMenu`? 😊 
```suggestion
  const shouldDisplayDesktopMenu = !useMediaQuery(MEDIA_QUERY_MAX_WIDTH);
  const shouldDisplayText = !shouldDisplaySmallMenu;

  return (
    <StudioPageHeader>
      <StudioPageHeader.Main>
        <StudioPageHeader.Left showTitle={shouldDisplayText} title={app} />
        {shouldDisplayDesktopMenu && (
          <StudioPageHeader.Center>
            {menuItems && (
              <LargeNavigationMenu
                menuItems={menuItems.map((menuItem: HeaderMenuItem) => ({
                  link: menuItem.link,
                  name: t(menuItem.key),
                  isBeta: menuItem.isBeta,
                }))}
              />
            )}
          </StudioPageHeader.Center>
        )}
```

Reply:
Great suggestion! 😄